src/style.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Consider adding general stylings and resets to the separate file to keep your main styles file clean